packages/terra-clinical-header/src/Header.jsx

```diff
@@ -60,30 +60,29 @@ const Header = ({
     );
   }
 
-  let endElement;
-  if (endContent) {
-    endElement = <div className={cx('flex-end')}>{endContent}</div>;
-  }
-
-  let childElement;
-  if (children) {
-    const childClassNames = cx([
-      'flex-collapse',
-      children.props.className,
-    ]);
-    childElement = React.cloneElement(children, { className: childClassNames });
-  }
+  const content = React.Children.map(children, child => (
+    React.cloneElement(child, {
+      className: classNames([cx('flex-collapse'), children.props.className]),
+    })
+  ));
 
-  const headerClass = isSubheader ? 'flex-subheader' : 'flex-header';
+  const headerClassNames = classNames([
```

Comment:
the classNames module will filter out any class that is undefined, which is why I am adding `customProps.className` directly. I used `classNames` instead of `cx` to prevent binding the custom class to our stylesheet.